code/modules/clothing/spacesuits/rig/suits/alien.dm

```diff
@@ -152,3 +152,39 @@
 
 	initial_modules = list(
 		)
+//ChompEdit Begins
+
+/obj/item/weapon/rig/vox/engineering
+	suit_type = "sinister alien"
+	icon_state = "voxstealth_rig"
+	desc = "A lightweight, alien rig dedicated for construction and engineering tasks. Not reccomended for hostile engagement."
+	armor = list (melee = 25, bullet = 5, laser = 40, energy = 45, bomb = 50, bio = 100, rad = 100) //CE suit values but shuffled to a tighter focus on the job hazards
+	flags = PHORONGUARD
+	item_flags = THICKMATERIAL
+	siemens_coefficient = 0.1
+	offline_slowdown = 2.5
+	slowdown = 0 
+	emp_protection = 40 //change this to 30 if too high.
+
+	req_one_access = list()
+	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank,/obj/item/device/suit_cooling_unit,/obj/item/weapon/storage)
+	offline_vision_restriction = 1
+
+	initial_modules = list(
+	)
+
+	air_type = /obj/item/weapon/tank/vox
+
+	max_heat_protection_temperature = FIRE_HELMET_MAX_HEAT_PROTECTION_TEMPERATURE
+
+/obj/item/weapon/rig/vox_engine_technition_control_module/equipped
```

Comment:
This inherits nothing from `/obj/item/weapon/rig/vox/engineering` but instead inherits all from `/obj/item/weapon/rig/vox`
https://github.com/CHOMPStation2/CHOMPStation2/blob/30843dfb81eb5a059ae81c91cda839488ad18147/code/modules/clothing/spacesuits/rig/suits/alien.dm#L57